src/Features/VisualBasic/Portable/FileHeaders/VisualBasicFileHeaderCodeFixProvider.vb

```diff
@@ -0,0 +1,33 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports Microsoft.CodeAnalysis.FileHeaders
+Imports Microsoft.CodeAnalysis.LanguageServices
+Imports Microsoft.CodeAnalysis.VisualBasic.LanguageServices
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.FileHeaders
+    <ExportCodeFixProvider(LanguageNames.VisualBasic, Name:=NameOf(VisualBasicFileHeaderCodeFixProvider))>
+    <[Shared]>
+    Friend Class VisualBasicFileHeaderCodeFixProvider
+        Inherits AbstractFileHeaderCodeFixProvider
+
+        Protected Overrides ReadOnly Property FileHeaderHelper As AbstractFileHeaderHelper
+            Get
+                Return VisualBasicFileHeaderHelper.Instance
+            End Get
+        End Property
+
+        Protected Overrides ReadOnly Property SyntaxKinds As ISyntaxKinds
+            Get
+                Return VisualBasicSyntaxKinds.Instance
+            End Get
+        End Property
+
+        Protected Overrides Function ParseLeadingTrivia(text As String) As SyntaxTriviaList
+            Return SyntaxFactory.ParseLeadingTrivia(text)
```

Comment:
nit fwiw SyntaxFacts has ParseToken, so it would be morally fine to have ParseLeadingTrivia there.